### ISSUE TITLE
All Travis CI builds failing

### ISSUE THREAD

**Issue by nerual** [railsbridge/bridge_troll#661]
Any sort of change (documentation, etc) is failing in Travis with similar errors:
```
bundle install --without development --without production --deployment
Traceback (most recent call last):
	4: from /home/travis/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `<main>'
	3: from /home/travis/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `eval'
	2: from /home/travis/.rvm/gems/ruby-2.5.3/bin/bundle:23:in `<main>'
	1: from /home/travis/.rvm/rubies/ruby-2.5.3/lib/ruby/site_ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/home/travis/.rvm/rubies/ruby-2.5.3/lib/ruby/site_ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
The command "eval bundle install --without development --without production --deployment " failed. Retrying, 2 of 3.```

**Comment by rachelober** [comment]
We believe this is the issue and mentions a fix.
https://bundler.io/blog/2019/05/14/solutions-for-cant-find-gem-bundler-with-executable-bundle.html

**Comment by nerual** [comment]
Rachel fixed this in #662. closing!
